src/renderer/services/chain/fees/stake.ts

```diff
@@ -20,7 +20,13 @@ import { reloadStakeFeesByChain } from './fees.helper'
 // `TriggerStream` to reload stake fees
 const { stream$: reloadStakeFees$, trigger: reloadStakeFees } = triggerStream()
 
-// reload fees
+/**
+ * reload fees
+ *
+ * Has to be used ONLY on an appopriate screen
+ * @example
+ * useSubscription(updateStakeFeesEffect$) - ONLY
```

Comment:
Helpful comment, too!

[ntipick] typo: `appropriate view` (just a missing `r`). Last `- ONLY` can be removed I think.